BUILD.gn

```diff
@@ -52,6 +51,11 @@ if (current_toolchain != "${dir_pw_toolchain}/dummy:dummy") {
       "${nlunit_test_root}:nlunit-test",
     ]
 
+    # Temporary until RNG issue sorted out
```

Comment:
should we add some TODO with github issue number to describe the RNG issue and add a fix?